src/http/api/v1/exports.ts

```diff
@@ -109,14 +109,23 @@ async function exportVoteCoefficientsCSV(db: JsonStorage, round: Round) {
     }),
   ]);
 
+  const usePassport =
```

Comment:
is it a react hook?

Reply:
@boudra  yeah it's defined server side but it runs on a remote browser.

Reply:
Mo is right! This should be conventionally boolean name like - such as `isPassport` or `isSybilDefended`, etc.

Reply:
@hmrtn @boudra  this var tells the rest of the program if we need to use passport or not. 
`isPassport` wouldn't make sense.

would you prefer `isPassportEnable`?

Reply:
`isPassportEnabled` sounds good

Reply:
I updated it to `isPassportEnable`